api/cmd/main.go

```diff
@@ -32,11 +32,13 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-const (
-	serviceName = "soda.multicloud.v1.api"
-)
-
 func main() {
+
+	serviceName := "api"
```

Comment:
Please use ```constants``` block to define constants.
Please also check rest of the code.

Reply:
done